src/main/java/net/openhft/chronicle/bytes/ByteStringAppender.java

```diff
@@ -395,5 +395,6 @@ default B append(@NotNull BigDecimal bigDecimal) {
     }
 
     // internal method to cache a byte[]
+    @Deprecated(/* to be removed in x.26 */)
```

Comment:
can we remove sooner? I vote for x.25 or even x.24

Reply:
This is used by a number of methods